Fix validate compilation issues due to removal of veraPDF artifacts from maven central

## 🐛 Describe the bug
<!-- A clear and concise description of what the bug is. -->
🤯 thought this was entirely not possible, but apparently not. veraPDF folks somehow removed artifacts from maven central for their previous version, so we have to upgrade to the latest.

From the GA runs (and tested locally):
```
Error: L:pds.roundup.util:🪵 Stdout = «[ERROR] Failed to execute goal on project validate: Could not resolve dependencies for project gov.nasa.pds:validate:jar:2.1.5-SNAPSHOT: The following artifacts could not be resolved: org.verapdf.pdfbox:pdfbox:jar:2.0.44, org.verapdf.pdfbox:preflight:jar:2.0.44: Could not find artifact org.verapdf.pdfbox:pdfbox:jar:2.0.44 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

## Acceptance criteria
If the software compiles, and baseline regression tests pass, we should be good.

--- a/pom.xml
+++ b/pom.xml
@@ -189,7 +189,7 @@ POSSIBILITY OF SUCH DAMAGE.
     <dependency>
       <groupId>org.verapdf</groupId>
       <artifactId>pdfbox-validation-model</artifactId>
-      <version>1.16.1</version>
+      <version>1.20.1</version>
     </dependency>
 
     <dependency>

